src/cpp/library/math.h

```diff
@@ -0,0 +1 @@
+#include <cmath>
```

Comment:
Why do we need "math.h" ?

Reply:
I investigated the `math.h` file from the C++ standard library, which has `cmath` as a header file. The header file used in our COM is `math.h`. Should we add a `math.h` file to avoid the error I showed in the description?